README.md

```diff
@@ -5,13 +5,17 @@
 This is a documentation site for [Synapse](https://www.synapse.org). Synapse is an open source software platform that data
 scientists use to carry out, track, and communicate their research in real time.
 
-## How to contribute
+## Contributing Guide
 
-Synapse Docs is generated using [Jekyll](https://jekyllrb.com/) and uses redcarpet to render Markdown. Various page layouts can be found under the \_layouts folder in the home directory. Most everything can be written using standard [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
+- Assign github issue to yourself to track revisions.
+- Create a [feature branch](https://guides.github.com/introduction/flow/) to make you revisions.
+- Use a Markdown editor (e.g. Typora) to visualize edits locally. 
+- Open a PR and request a review.
+- Merge requires approving review by product manager or ....
 
-Internal development can be performed by branching from `develop` to your own feature branch, making changes, pushing the branch to this repository, and opening a pull request. Pull requests against the master branch require a review before merging.
+Synapse Docs is generated using [Github Pages](https://pages.github.com/). Follow the standard [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) guide.
```

Comment:
```suggestion
Synapse Docs is generated using [Github Pages](https://pages.github.com/). Follow the standard [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) guide.

To build locally, follow the instructions found [here](https://help.github.com/en/articles/testing-your-github-pages-site-locally-with-jekyll). You will need Jekyll, Ruby, and the Ruby package manager, Bundler.

```

Reply:
Does it make sense to clarify that building locally is not required as a contributor can check format in another Markdown editor? It might be a more inclusive workflow.

Reply:
O I see that is addressed below :)!